src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+
+    public String getRandomBall() {
+        Random random = new Random();
+        Ball ball = new Ball();
+        ball.setNumber(random.nextInt(100));
```

Comment:
All magic numbers in your code should be constants.

see https://mate-academy.github.io/jv-program-common-mistakes/java-core/oop/lottery